package.json

```diff
@@ -81,7 +81,6 @@
     "eslint-plugin-import": "^2.26.0",
     "eslint-plugin-prettier": "^4.0.0",
     "gulp": "^4.0.2",
-    "gulp-typescript": "^5.0.1",
```

Comment:
Did you accidentally remove it? Could you bring it back?

Reply:
Fixed